Makefile

```diff
@@ -1,3 +1,9 @@
+# Copyright (C) 2020  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
```

Comment:
Please add a `SPDX` specifiers as described in https://github.com/SymbiFlow/ideas/issues/47